<launch>
    <node pkg="marker_detector" type="aruco_detector" name="aruco_detector" output="screen">
        <param name= "cam_cal_file" type="string" value="$(find marker_detector)/config/left.yaml"/>
        <remap from="~image_raw" to="/camera/color/image_raw"/>
    </node>
</launch>
